Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpBuildDeployPreferences.cs

```diff
@@ -3,13 +3,14 @@
 
 using Microsoft.MixedReality.Toolkit.Utilities.Editor;
 using Microsoft.MixedReality.Toolkit.WindowsDevicePortal;
+using System;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.Build.Editor
 {
     public static class UwpBuildDeployPreferences
     {
-        public const string MIN_SDK_VERSION = "10.0.17134.0";
+        public static Version MIN_SDK_VERSION = new Version("10.0.17134.0");
```

Comment:
Shouldn't this be whatever version of the insider preview you guys are forcing people to use for mrtk?

Reply:
Yes, though this intentionally wasn't changed in this PR to keep the change scoped to changing the version-enumeration code.